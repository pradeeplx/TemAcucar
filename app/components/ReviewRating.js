import React from 'react'
import { View } from 'react-native'
import Colors from "../Colors"
import Icon from "./Icon"

export default ReviewRating = ({ rating }) => (
  <View style={{
    flexDirection: 'row',
    marginBottom: 4,
  }}>
    { [1, 2, 3, 4, 5].map((index) => (
      <Icon key={index} name={(index > rating ? "ios-star-outline" : "ios-star")} style={{
        color: Colors.darkYellow,
      }}/>
    )) }
  </View>
)
