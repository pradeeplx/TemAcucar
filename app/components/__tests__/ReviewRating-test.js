import 'react-native'
import React from 'react'
import ReviewRating from '../ReviewRating'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <ReviewRating rating={4} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
