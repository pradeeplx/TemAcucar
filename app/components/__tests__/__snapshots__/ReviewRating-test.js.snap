exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "marginBottom": 4,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Ionicons",
          "fontSize": 27,
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "color": "#FFBA00",
        },
      ]
    }>
    
  </Text>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Ionicons",
          "fontSize": 27,
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "color": "#FFBA00",
        },
      ]
    }>
    
  </Text>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Ionicons",
          "fontSize": 27,
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "color": "#FFBA00",
        },
      ]
    }>
    
  </Text>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Ionicons",
          "fontSize": 27,
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "color": "#FFBA00",
        },
      ]
    }>
    
  </Text>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Ionicons",
          "fontSize": 27,
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "color": "#FFBA00",
        },
      ]
    }>
    
  </Text>
</View>
`;
